### ISSUE TITLE
L2_reg is missing

### ISSUE THREAD

**Issue by elstonsa** [harvitronix/five-video-classification-methods#134]
It seems like L2_reg is missing causing lrcn model to fail

**Comment by jaesungryu96** [comment]
maybe you can fix that problem with import

from keras.regularizers import l2 as L2_reg

**Comment by elstonsa** [comment]
Yes. but do you know if L2_reg == keras.regularizers.l2 / or some custom function?

**Comment by jaesungryu96** [comment]
sorry I don't know about that

**Comment by alexcosmin** [comment]
@elstonsa did you ever find out if L2_reg == keras.regularizers.l2 or if it was a custom function?
